megamek/src/megamek/client/ui/swing/GUIPreferences.java

```diff
@@ -485,6 +518,16 @@ protected GUIPreferences() {
         store.setDefault(MINI_REPORT_POS_Y, 150);
         store.setDefault(MINI_REPORT_SIZE_HEIGHT, 300);
         store.setDefault(MINI_REPORT_SIZE_WIDTH, 400);
+        store.setDefault(MINI_REPORT_ENABLED, true);
+        store.setDefault(MINI_REPORT_AUTO_DISPLAY_REPORT_PHASE, MSG_SHOW);
```

Comment:
Repeated from above: This should be boolean values, unless there is a reason against that that Im not seeing. At the very least it shouldnt be localized strings.